python/ray/rllib/agent.py

```diff
@@ -195,6 +197,34 @@ def save(self):
             open(checkpoint_path + ".rllib_metadata", "wb"))
         return checkpoint_path
 
+    def save_to_object(self):
+        """Saves the current model state to the Ray object store.
+
+        Returns:
+            ObjectID of the checkpoint.
+        """
+
+        checkpoint_prefix = self.save()
```

Comment:
this is a write to disk; does it make sense to store this checkpoint prefix in the agent as a checkpoint path too (as a fallback)? We would reorder the restoring

Reply:
Writes to disk are fast sine in the common case you hit fs cache.